pkg/web_app/lib/src/account.dart

```diff
@@ -340,6 +340,10 @@ class _PkgAdminWidget {
   Future<void> _setRetracted() async {
     final version =
         materialDropdownSelected(_retractPackageVersionInput)?.trim() ?? '';
+    if (version.isEmpty) {
+      await _validateVersionSelection();
+    }
```

Comment:
shouldn't this `return;`?

Reply:
Yes of course. I was too quick when extracting the helper method!